### DISCUSSION TITLE
DNS Setup: Resolve SPF Redirect records

### DISCUSSION THREAD

**Discussion by sphr2k** [postalserver/postal#1815] [Feature suggestions]
In DNS setup, SPF Redirect records are not resolved correctly. Example:

![image](https://user-images.githubusercontent.com/1213706/149911452-d2ede647-a503-4cdd-a299-9ee048a55f36.png)

![image](https://user-images.githubusercontent.com/1213706/149912258-34ab1558-af10-4881-a343-b08e410a4563.png)

**Comment by willpower232** [comment]
Yes I don't think it does any following of records, just a basic check on the first record it finds. I don't think this prevents Postal from sending your messages if you can ignore the warning.
